### DISCUSSION TITLE
Potential Problem - Proxmox Backup Server post install issue

### DISCUSSION THREAD

**Discussion by Damien455** [tteck/Proxmox#3258] [Help Needed] [answered]
Hi all, installed proxmox backup server on debian 12 standard lxc (instructions here https://forum.proxmox.com/threads/proxmox-backup-server-on-lxc-with-external-nas.79489/)  ran the helper script (thank you there amazing use it on all my proxmox servers) for the first for the backup server and it seem to crash out
                                                                                                                                   
 - Updating Proxmox Backup Server (Patience)...root@prox-backup:~# 

dont get any success message or other questions.

if i try an apt update i get the following errors

Hit:1 http://security.debian.org/debian-security bookworm-security InRelease
Hit:2 http://deb.debian.org/debian bookworm InRelease        
Hit:3 http://deb.debian.org/debian bookworm-updates InRelease
Get:4 http://download.proxmox.com/debian/pbs bookworm InRelease [2782 B]
Get:5 http://download.proxmox.com/debian/pve bullseye InRelease [2768 B]
Err:4 http://download.proxmox.com/debian/pbs bookworm InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 1140AF8F639E0C39
Err:5 http://download.proxmox.com/debian/pve bullseye InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY DD4BA3917E23BF59
Reading package lists... Done
W: GPG error: http://download.proxmox.com/debian/pbs bookworm InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 1140AF8F639E0C39
E: The repository 'http://download.proxmox.com/debian/pbs bookworm InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
W: GPG error: http://download.proxmox.com/debian/pve bullseye InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY DD4BA3917E23BF59
E: The repository 'http://download.proxmox.com/debian/pve bullseye InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
root@prox-backup:~# 

any suggestions please

**Comment by tteck** [comment, marked as answer]
Your sources are all jacked up! You have PVE, PBS, Bullseye and Bookworm. I'd suggest using https://tteck.github.io/Proxmox/#proxmox-backup-server-lxc then https://tteck.github.io/Proxmox/#proxmox-backup-server-post-install

**Comment by tteck** [comment]
The guide you followed is four years old!!
